features/list_artifact_files.feature

```diff
@@ -14,4 +14,4 @@ Feature: Listing files within an artifact
   Scenario: Try to list files without permission
     Given I have no role
     When I ask for a list of files in the artifact
-    Then my request is denied
+    Then my request for a list of files is denied
```

Comment:
This is the particular reason I changed all these related scenario warnings. `my request is denied` would be too vague for me to define a step.

Reply:
Can't do `last_response`, but I replaced all my main api responses with `@api_response`, so this is still workable that way.